pallets/runtime/tests/src/settlement_test.rs

```diff
@@ -1937,45 +1960,46 @@ fn basic_fuzzing() {
                 ));
             }
 
+            // Get state of locked tokens
+            let mut locked_assets = HashMap::new();
+            for user in &users {
+                locked_assets.insert(
+                    user.did,
+                    <PortfolioLockedAssets<TestStorage>>::iter_prefix(
+                        PortfolioId::default_portfolio(user.did),
+                    )
+                    .collect::<Vec<_>>(),
+                );
+            }
+
```

Comment:
Shouldn't this test be doing is own tracking of which assets should be locked, instead of just grabbing the locked assets balances from the chain-state?

Also this is getting the locked assets after the `withdraw_affirmation` call.  Shouldn't the test be comparing the locked asset before and after the `withdraw_affirmation` call?